share/functions/fish_clipboard_copy.fish

```diff
@@ -0,0 +1,7 @@
+function fish_clipboard_copy
+	if type -q pbcopy
```

Comment:
Yeah, this isn't performance-sensitive at all, and the bulk of the time is likely to be spent in the called command (`xsel` needs a trip to the X server and back).

Plus it would mean what you get via `funced` differs from the contents of the file and I like being able to use that for development. It's possible there's more tools to support here (there's also `xclip`, but I don't know if anything has it but not xsel), so it'd be nice to leave that possibility open.